addons/web_editor/static/src/js/backend/field_html.js

```diff
@@ -192,7 +192,10 @@ var FieldHtml = basic_fields.DebouncedField.extend(TranslatableFieldMixin, {
                 <div class="d-flex flex-column">
                     <div class="d-flex flex-row overflow-auto">
                         <t t-zone="main_sidebar"/>
-                        <div class="d-flex flex-column overflow-auto o_editor_center">
+                        <div class="d-flex flex-column overflow-auto o_editor_center">                            
+                            <div class="o_toolbar">
+                                <t t-zone="tools"/>
+                            </div>
```

Comment:
If we only have the floating menu bar, we cannot, for example, insert an image at a specific position, because we need a range to open the floating menu bar.
So for now I have put back the normal menu bar.